library/scrollphathd/api/http.py

```diff
@@ -0,0 +1,79 @@
+import scrollphathd
+from scrollphathd.fonts import font3x5
+
+try:
+    from queue import Queue
+except ImportError:
+    from Queue import Queue
+
+from .action import Action
+from .stoppablethread import StoppableThread
+from http import HTTPStatus
+
+try:
+    from flask import Blueprint, render_template, abort, request
+except ImportError:
+    raise ImportError("flask must be installed in order to use the api. Install with pip install flask")
+
+scrollphathd_blueprint = Blueprint('scrollhat', __name__)
+api_queue = Queue()
+
+@scrollphathd_blueprint.route('/scroll', methods=["POST"])
+def scroll():
+    data = request.get_json()
+    try:
+        api_queue.put(Action("scroll", (data["x"], data["y"])))
+    except KeyError:
+        return {"error": "keys x and y not posted."}, HTTPStatus.UNPROCESSABLE_ENTITY.value
```

Comment:
Flask isn't clever enough to automatically and implicitly jsonify a raw dictionary into a sane response (not sure if this is to protect people from themselves, or if I've got an older version) but it can be done explicitly with `flask.jsoninfy()`